private/protocol/rest/unmarshal.go

```diff
@@ -14,8 +14,8 @@ import (
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/awserr"
-	"github.com/aws/aws-sdk-go/aws/awsutil"
 	"github.com/aws/aws-sdk-go/aws/request"
+	awsStrings "github.com/aws/aws-sdk-go/internal/strings"
```

Comment:
Nit: Can we define a nomenclature for import alias. 
My first impression of `awsStrings` was it is an aws utility and should have been aliased as awsutil (which is not the case here). The name `awsStrings` doesn't tell me that this is an internal utility.

I would prefer something like `internalUtil` for internal utility imports.